pyouroboros/dockerclient.py

```diff
@@ -83,6 +83,13 @@ def stop(self, container):
             container.stop()
 
     def remove(self, container):
+        if container.attrs['HostConfig']['AutoRemove']:
+            self.logger.debug('Not removing container "%s" since autoremove was set during start.', container.name)
+            try:
+                container.wait(condition='removed')
```

Comment:
I like the idea of ignoring run_once containers with `pass`, but calling a wait() is a blocking function that could sit indefinitely ya know?

Maybe we check it in the update section instead of during remove?

Reply:
That's true. I'm assuming there that a container.stop() was executed beforehand (which `SIGKILL`s containers after 10 seconds), and then it shouldn't happen that `wait()` doesn't terminate in reasonable time as far as I understand.  
We could also specify a timeout in the call of `wait()`, and skip the container if it fires?

Reply:
right but what im saying is we have let a LOT happen (including updating said container) to a run once. If its a run once we shouldnt even be updating it! we should let it lie.

Reply:
Now I get your point. You're assuming that containers with `--rm` set are "short-term foreground processes" as the docker documentation states. In this case we probably shouldn't touch these containers at all (as you already said).  
I was confused, because technically it's also possible to start a detached container with the `--rm` flag, thus this isn't a perfect indicator if a container should only "run once".  
If you want me to modify the PR so that these containers are completely ignored by ouroboros just give me a short confirmation.

Reply:
i say yeah do it solely because docker defines it as such so we should too i think.

Reply:
but with a logger during passes in debug saying something along the lines of skipped because —rm set etc